app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -131,6 +140,90 @@ class TopicPlayFragmentTest {
     intended(hasExtra(StoryActivity.STORY_ACTIVITY_STORY_ID_ARGUMENT_KEY, storyId))
   }
 
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_chapterListIsNotVisible() {
+    activityTestRule.launchActivity(null)
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_recycler_view)).check(
+      matches(
+        not(
+          isDisplayed()
+        )
+      )
+    )
+  }
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIcon_iconChanges() {
+    activityTestRule.launchActivity(null)
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_list_view_control)).check(
+      matches(
+        withDrawable(R.drawable.ic_keyboard_arrow_down_black_24dp)
+      )
+    )
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_list_view_control)).perform(click())
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_list_view_control)).check(
+      matches(
+        withDrawable(R.drawable.ic_keyboard_arrow_up_black_24dp)
+      )
+    )
+  }
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIcon_chapterListIsVisible() {
+    activityTestRule.launchActivity(null)
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_list_view_control)).perform(click())
+    onView(
+      atPositionOnView(
+        R.id.story_summary_recycler_view,
+        0,
+        R.id.chapter_recycler_view
+      )
+    ).check(matches(isDisplayed()))
+  }
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_clickChapter_opensExplorationActivity() {
+    activityTestRule.launchActivity(null)
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 1, R.id.chapter_list_view_control)).perform(click())
+    onView(
+      allOf(
+        withId(R.id.chapter_recycler_view),
+        withParent(
+          atPosition(R.id.story_summary_recycler_view, 1)
+        )
+      )
+    ).check(matches(hasDescendant(withText(containsString("Second"))))).perform(click())
+    intended(hasComponent(ExplorationActivity::class.java.name))
+  }
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIconIndex0_clickExpandListIconIndex1_chapterListForIndex0IsNotDisplayed() {
```

Comment:
It might be worth also adding a test to show that the chapter list for index 1 can be displayed with the correct elements (it complements this test nicely since together they verify that index 1's list is different than index 0's).

Reply:
Added another test case 
`testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIconIndex1_clickExpandListIconIndex0_chapterListForIndex0IsNotDisplayed`